sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -369,6 +369,37 @@ object Radar {
         })
     }
 
+    /**
+     * Gets the device's current location with the desired accuracy.
+     *
+     * @param[desiredAccuracy] The desired accuracy.
+     * @param[callback] An optional callback.
+     */
+    @JvmStatic
```

Comment:
nit: not sure if we need `JvmStatic`?

Reply:
We do, otherwise `Radar.getLocation()` won't be exposed in Java (would need to call `Radar.INSTANCE.getLocation()`)